runtime/cudaq/platform/quantum_platform.cpp

```diff
@@ -146,6 +146,12 @@ void quantum_platform::onRandomSeedSet(std::size_t seed) {
   for (auto &qpu : platformQPUs)
     qpu->onRandomSeedSet(seed);
 }
+
+void quantum_platform::resetLogStream() { platformLogStream = nullptr; }
+std::ostream *quantum_platform::getLogStream() { return platformLogStream; }
+void quantum_platform::setLogStream(std::ostream &logStream) {
```

Comment:
```suggestion
void quantum_platform::resetLogStream() { platformLogStream = nullptr; }

std::ostream *quantum_platform::getLogStream() { return platformLogStream; }

void quantum_platform::setLogStream(std::ostream &logStream) {
```
nit: Make a bit more readable with whitespace.